src/main/kotlin/org/kamiblue/client/module/modules/combat/AutoOffhand.kt

```diff
@@ -109,11 +111,11 @@ internal object AutoOffhand : Module(
 
             updateDamage()
 
-            if (!movingTimer.tick(200L, false)) return@safeListener // Delays 4 ticks by default
+            if (!movingTimer.tick(delay.toLong(), false)) return@safeListener // Delays `delay` ticks
 
             if (!player.inventory.itemStack.isEmpty) { // If player is holding an in inventory
                 if (mc.currentScreen is GuiContainer) {// If inventory is open (playing moving item)
```

Comment:
space before `//` please

Reply:
There is already a space here from the previous comment.

Reply:
![Screenshot_20210207-234250.png](https://user-images.githubusercontent.com/17222512/107176736-38957400-699e-11eb-8317-c6d69ae054c3.png)

Reply:
I'll add the line, but I didn't modify that comment :^)